include/zephyr/data/jwt.h

```diff
@@ -24,10 +24,10 @@ extern "C" {
 /**
  * @brief JWT data tracking.
  *
- * JSON Web Tokens contain several sections, each encoded in base-64.
+ * JSON Web Tokens contain several sections, each encoded in Base64URL.
  * This structure tracks the token as it is being built, including
  * limits on the amount of available space.  It should be initialized
- * with jwt_init().
+ * with \ref jwt_init_builder().
```

Comment:
nit but please use `@` instead of `\`s for Doxygen commands.
In that particular instance, you may actually drop the `@ref` altogether as Doxygen will automagically resolve the reference

Reply:
```suggestion
 * with jwt_init_builder().
```